nxdrive/direct_edit.py

```diff
@@ -386,10 +387,14 @@ def _prepare_edit(
         file_path = dir_path / filename
 
         # Download the file
-        tmp_file = self._download(engine, info, file_path, blob, xpath, url=url)
-        if tmp_file is None:
-            log.error("Download failed")
-            return None
+        FileAction("Download", file_path, size=0, reporter=QApplication.instance())
+        try:
+            tmp_file = self._download(engine, info, file_path, blob, xpath, url=url)
+            if tmp_file is None:
+                log.error("Download failed")
```

Comment:
As we have no information about the error, could it be a warning to not pollute Sentry envents?